otterdog/eclipse-sw360.jsonnet

```diff
@@ -10,11 +10,19 @@ local customRuleset(name) =
     ],
     required_pull_request+: {
       required_approving_review_count: 1,
-      requires_last_push_approval: true,
-      requires_review_thread_resolution: true,
       dismisses_stale_reviews: true,
+      requires_code_owner_reviews: true,
+      requires_last_push_approval: true,
+      requires_review_thread_resolution: true
     },
+    required_approving_review_count: 1,
     requires_linear_history: true,
+    requires_conversation_resolution: true,
+    dismisses_stale_reviews: true,
+    bypass_force_push_allowances+: [
```

Comment:
its slightly different for rulesets, but the same setting is already there above.

Reply:
Done